render/marquee.go

```diff
@@ -73,7 +74,7 @@ func (m Marquee) FrameCount() int {
 		offend = -cw
 	}
 
-	return cw + offstart + size - offend + 1
+	return cw + offstart + size - offend
```

Comment:
Fixed the extra "jump" in `render.Marquee` that some people were complaining about.